i18n/generate.py

```diff
@@ -175,6 +175,8 @@ def run(self, args):
         # Dummy text is not required. Don't raise exception if files are missing.
         for locale in config.CONFIGURATION.dummy_locales:
             merge_files(locale, fail_if_missing=False)
+        # Merge the source locale, so we have the canonical .po files.
+        merge_files(config.CONFIGURATION.source_locale, fail_if_missing=args.strict)
```

Comment:
Will it merge twice if ``source_locale`` is also in ``langs``? Can that happen, and if so is it even a problem? Maybe it should only do this line if ``source_locale`` is not in ``langs``.